public/js/projects.js

```diff
@@ -0,0 +1,95 @@
+const refnotxt = document.getElementById('refnotxt');
+const projectnametxt = document.getElementById('projectnametxt');
+const sectortxt = document.getElementById('sectortxt');
+const statusselect = document.getElementById('statusselect');
+const locationtxt = document.getElementById('locationtxt');
+const projectstablebody = document.getElementById('projectstablebody');
+const modalconfirmdelete = document.getElementsByClassName('confirmdelete')[0];
+
+function getFilteredProjects(cb) {
+  const filterparams = {
+    refno: refnotxt.value,
+    projectname: projectnametxt.value,
+    sector: sectortxt.value,
+    coordinationstatusid: statusselect.value,
+    location: locationtxt.value,
+  };
+  request('POST', '/getprojects', filterparams, cb);
+}
+
+function renderFitchedProjects(projects) {
+  projectstablebody.innerHTML = '';
```

Comment:
Use .textContent here and avoid .innerHTML unless it's extremely necessary and doesn't leave a security risk.

Reply:
done